README.md

```diff
@@ -0,0 +1,58 @@
+# EdgeCast CLI
+
+This is the official command line interface (CLI) for EdgeCast.
+
+## Getting Started
+
+### Requirements
+- Go 1.17
+
+### Installation
+To install the CLI, use the makefile:
+```
+$ make install
+```
+
+### Using the CLI
+Commands follow the following format:
```

Comment:
We haven't got there yet. But we should list all available commands, their sub-commands, and flags...

Reply:
Yeah, I just didn't want to leave a TODO here.